### ISSUE TITLE
A11y_SQL Azure Data Studio_Connect to a SQL Server_AI4D: Name is not defined for "New Connection" and "New Server Group" button in the "Servers" section.

### ISSUE THREAD

**Issue by PadminiGariki** [dotnet/maui#16812]
### Description
[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Aug2023; #SQL Azure Data Studio; #WCAG4.1.2; #AI4D; #Win32; #Benchmark; #FTP;

### Environment Details:​
Application Name: Sql Azure Data studio
Version: 1.45.1
OS: Win11

### Steps to Reproduce

1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon and select the 'New Server Group' icon.
4. Observe Name is not defined for "New Connection" and "New Server Group" button in the "Servers" section.

### Actual Results:
Name is not defined for "New Connection" button in the "Servers" section.

**Similar issue is also reproducing in the below scenarios:**
**1.Issue: Export query results** 
1.Launch Azure Data Studio.
2.Navigate to File > New Query
3.Connect to a Server in the Query Editor and choose the credential, and optionally the database(DW Configuration) to connect to
4.Type a SQL query statement that produces a result set.
5.Click Run
6.Navigate to the Results tab. 
7.Select a cell (or cell areas) then right click. 
8.From the right click menu, choose "Save As <file type>" , where <file type> is either CSV, Excel, JSON, XML. 
9.Observe Name is not defined for "Save as" button.

**2.Issue:Notebooks - Code cell**
1.Launch Azure Data Studio Insiders application.
2.Connect to server.
3.Right-click connection and click New Notebook
4.Click on cell and select code cell.
5. name is not defined for "Edit", "Move cell down", "Move cell up" and "Delete" buttons in title bar of "code cell".

### Expected Results:
Name should be defined for "New Connection" and "New Server Group" button in the "Servers" section.

### User Impact:
screen reader user will not be able to know the name of the "New Connection" and "New Server Group" button in the "Servers" section. If it is not defined has name.

### Attachment:
<img width="960" alt="Name is not defined for New Connection button" src="https://github.com/dotnet/maui/assets/142479541/8297c8b7-3e89-4700-9e02-46adf1ecd8a9">

<img width="960" alt="similar issue for New server group button" src="https://github.com/dotnet/maui/assets/142479541/81dd8425-d94a-4c69-91c5-ba8ce8d47284">

<img width="960" alt="Similar issue save as" src="https://github.com/dotnet/maui/assets/142479541/10912044-3d0e-4abf-b33c-b0bec672078f">

<img width="960" alt="16 similar issue" src="https://github.com/dotnet/maui/assets/142479541/705165bd-4554-4116-ae10-05843ec637fe">

### Version with bug

7.0.92

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

**Comment by PureWeen** [comment]
Wrong Repository

**Comment by PadminiGariki** [comment]
Sorry Inconvenience caused. Adding #Closed; tag. Will file issue on Correct Repo.
